tests/test_suite_common.sh

```diff
@@ -443,6 +464,9 @@ compare_templates() {
     record=$3
     rc=0
     for f in `ls /tests/compiled_templates/${testdir}`; do
+	if [ $f = step2 ]; then
```

Comment:
Indentation looks off here. Also worth a comment on why we can skip this for step2?